docs/application/dotnet/get-started/first-app.md

```diff
@@ -259,6 +262,54 @@ To deploy and run the application on the emulator:
     </div>
 </div>
 
+## Debug your application in emulator
+
+The following example shows how how you can use the Tizen emulator to debug a basic Tizen .NET application.
```

Comment:
```suggestion
The following example shows how you can use the Tizen emulator to debug a basic Tizen .NET application.
```

Reply:
> 

Hello, @dkson95 thanks for reviewing.

FYI :  
application/vscode-ext/Tizen/dotnet.md ([PR 1608](https://github.com/Samsung/tizen-docs/pull/1608)) and application/vstools/Tizen/dotnet.md ([PR 1612](https://github.com/Samsung/tizen-docs/pull/1612))  are updated, 
so as I mailed to you, we think it would be hard to merge them to get-started/first-app.md in short time.
For this time we are focusing on restructuring ToC & Tizen Studio release,
and will do merging to  get-started/first-app.md later. Please check this.

Reply:
> > 
> 
> Hello, @dkson95 thanks for reviewing.
> 
> FYI : application/vscode-ext/Tizen/dotnet.md ([PR 1608](https://github.com/Samsung/tizen-docs/pull/1608)) and application/vstools/Tizen/dotnet.md ([PR 1612](https://github.com/Samsung/tizen-docs/pull/1612)) are updated, so as I mailed to you, we think it would be hard to merge them to get-started/first-app.md in short time. For this time we are focusing on restructuring ToC & Tizen Studio release, and will do merging to get-started/first-app.md later. Please check this.

Okay, agreed on the suggestion. 

@mijin85cho @asm-foysal  thank you to setup this overall re-structure well 👍 
I'd appreciated if you could please arrange remaining tasks to be proceed further for the contents update along with these restructuring .